src/components/Toggle/index.css

```diff
@@ -0,0 +1,141 @@
+.react-toggle {
+  touch-action: pan-x;
```

Comment:
weird. Haven't seen this before.

Reply:
This is a vendor'd library that we have to import temporarily. I had never seen it before either.